src/test/java/org/mockitousage/basicapi/MockingDetailsTest.java

```diff
@@ -1,54 +0,0 @@
-/*
- * Copyright (c) 2007 Mockito contributors
- * This program is made available under the terms of the MIT License.
- */
-package org.mockitousage.basicapi;
-
-import org.junit.Test;
-import org.mockito.Mock;
-import org.mockito.Mockito;
-import org.mockito.Spy;
-import org.mockitoutil.TestBase;
-
-import static junit.framework.TestCase.assertFalse;
-import static junit.framework.TestCase.assertTrue;
-import static org.mockito.Mockito.*;
-
-public class MockingDetailsTest extends TestBase {
```

Comment:
Actually I liked it as it allows to drive the user public API design and they can serve as _acceptance tests_ too. While other tests are better suited for unit tests. In this case there's a single class, but the distinction still apply since it's a public API.